No compiling possible with MKR1010WIFI and rgb shield

In line 25 of the .cpp file `SAMD` should be added after `SPIClass`:

```cpp
static SPIClassSAMD SPI_MATRIX(&sercom0, A3, A4, A3, SPI_PAD_0_SCK_1, SERCOM_RX_PAD_0);
```

With compliments of and to; J-M-L Jackson.

Funny... Just got my boards today and hit the same problem.  I also found [@janvol1 's post on Arduino forum](https://forum.arduino.cc/t/mkrwifi1010-and-rgb-shield/928253/16)


Here is the compilation log for the records: 

```
/home/andre/Arduino/libraries/Arduino_MKRRGB/src/MKRRGBMatrix.cpp:25:82: error: no matching function for call to 'arduino::HardwareSPI::HardwareSPI(SERCOM*, const uint8_t&, const uint8_t&, const uint8_t&, SercomSpiTXPad, SercomRXPad)'
 static SPIClass SPI_MATRIX(&sercom0, A3, A4, A3, SPI_PAD_0_SCK_1, SERCOM_RX_PAD_0);
                                                                                  ^
In file included from /home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/libraries/SPI/SPI.h:24:0,
                 from /home/andre/Arduino/libraries/Arduino_MKRRGB/src/MKRRGBMatrix.cpp:20:
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:104:7: note: candidate: constexpr arduino::HardwareSPI::HardwareSPI()
 class HardwareSPI
       ^~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:104:7: note:   candidate expects 0 arguments, 6 provided
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:104:7: note: candidate: constexpr arduino::HardwareSPI::HardwareSPI(const arduino::HardwareSPI&)
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:104:7: note:   candidate expects 1 argument, 6 provided
/home/andre/Arduino/libraries/Arduino_MKRRGB/src/MKRRGBMatrix.cpp:25:17: error: cannot declare variable 'SPI_MATRIX' to be of abstract type 'arduino::HardwareSPI'
 static SPIClass SPI_MATRIX(&sercom0, A3, A4, A3, SPI_PAD_0_SCK_1, SERCOM_RX_PAD_0);
                 ^~~~~~~~~~
In file included from /home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/libraries/SPI/SPI.h:24:0,
                 from /home/andre/Arduino/libraries/Arduino_MKRRGB/src/MKRRGBMatrix.cpp:20:
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:104:7: note:   because the following virtual functions are pure within 'arduino::HardwareSPI':
 class HardwareSPI
       ^~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:109:21: note: 	virtual uint8_t arduino::HardwareSPI::transfer(uint8_t)
     virtual uint8_t transfer(uint8_t data) = 0;
                     ^~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:110:22: note: 	virtual uint16_t arduino::HardwareSPI::transfer16(uint16_t)
     virtual uint16_t transfer16(uint16_t data) = 0;
                      ^~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:111:18: note: 	virtual void arduino::HardwareSPI::transfer(void*, size_t)
     virtual void transfer(void *buf, size_t count) = 0;
                  ^~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:114:18: note: 	virtual void arduino::HardwareSPI::usingInterrupt(int)
     virtual void usingInterrupt(int interruptNumber) = 0;
                  ^~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:115:18: note: 	virtual void arduino::HardwareSPI::notUsingInterrupt(int)
     virtual void notUsingInterrupt(int interruptNumber) = 0;
                  ^~~~~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:116:18: note: 	virtual void arduino::HardwareSPI::beginTransaction(arduino::SPISettings)
     virtual void beginTransaction(SPISettings settings) = 0;
                  ^~~~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:117:18: note: 	virtual void arduino::HardwareSPI::endTransaction()
     virtual void endTransaction(void) = 0;
                  ^~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:120:18: note: 	virtual void arduino::HardwareSPI::attachInterrupt()
     virtual void attachInterrupt() = 0;
                  ^~~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:121:18: note: 	virtual void arduino::HardwareSPI::detachInterrupt()
     virtual void detachInterrupt() = 0;
                  ^~~~~~~~~~~~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:123:18: note: 	virtual void arduino::HardwareSPI::begin()
     virtual void begin() = 0;
                  ^~~~~
/home/andre/.arduino15/packages/arduino/hardware/samd/1.8.12/cores/arduino/api/HardwareSPI.h:124:18: note: 	virtual void arduino::HardwareSPI::end()
     virtual void end() = 0;
                  ^~~
exit status 1
Error compiling for board Arduino MKR1000.
```

--- a/src/MKRRGBMatrix.cpp
+++ b/src/MKRRGBMatrix.cpp
@@ -22,7 +22,7 @@
 
 #include "MKRRGBMatrix.h"
 
-static SPIClass SPI_MATRIX(&sercom0, A3, A4, A3, SPI_PAD_0_SCK_1, SERCOM_RX_PAD_0);
+static SPIClassSAMD SPI_MATRIX(&sercom0, A3, A4, A3, SPI_PAD_0_SCK_1, SERCOM_RX_PAD_0);
 
 // This table is based on the formula: gamma = (int)(pow(i / 255.0, gamma) * 255 + offset)
 // where gamma = 2.5 and offset is 0.5

